src/metabase/async/streaming_response.clj

```diff
@@ -138,15 +154,15 @@
   immediately, returning `0`."
   [^Request request]
   (try
-    (let [^SocketChannel channel (.. request getHttpChannel getEndPoint getTransport)
-          buf    (ByteBuffer/allocate 1)
-          status (.read channel buf)]
+    (log/fatal "--------------------------------------------------")
```

Comment:
What is the thinking of this `log/fatal` here?

Reply:
. o O (Forgot to remove it)